src/main/python/cell_feature_page.py

```diff
@@ -0,0 +1,198 @@
+""" A page for displaying grouped key-value pairs describing feature 
+extraction results.
+"""
+from typing import Optional, Dict, List, Callable, Any
+
+from PyQt5.QtWidgets import (
+    QWidget, QVBoxLayout, QLayoutItem, QFrame, QScrollArea
+)
+
+from fx_data import FxData
+from flow_layout import FlowLayout
+from cell_feature_view import CellFeatureView
+
+
+KeysType = Dict[str, List[str]]
+
+
+def default_keys() -> KeysType:
+    return {
+        "general": [
+            "adaptation",
+            "avg_isi",
+            "blowout_mv",
+            "electrode_0_pa",
+            "f_i_curve_slope",
+            "rheobase_sweep_num",
+            "ri",
+            "sag",
+            "seal_gohm",
```

Comment:
"seal_gohm" is also a resistance. Perhaps, it makes sense to combine "resistance" and "general"